src/crimson/osd/objclass.cc

```diff
@@ -126,6 +126,19 @@ int cls_cxx_stat2(cls_method_context_t hctx,
                   uint64_t *size,
                   ceph::real_time *mtime)
 {
+  assert(size);
+  assert(mtime);
```

Comment:
https://github.com/ceph/ceph/pull/42576#discussion_r682532214

please reconsider my comment.

Reply:
I dropped the assert. See my reply.